packages/shared/src/lib/func.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { shuffle } from './func';
+
+describe('shuffle function', () => {
+  it('should shuffle the array', () => {
+    const array = [1, 2, 3, 4, 5];
+    const arrayCopy = [...array];
+    shuffle(array);
+    expect(array).not.toEqual(arrayCopy);
```

Comment:
Technically it could return the same as array right? (making it potentially flaky)

Reply:
Yes but what are the odds 🤔 ... any ideas to not have a flaky test? or should we remove it?

Reply:
We could use a random seed in the `Math.random` function. That way, the test is always repeatable.